DOL.WHD.Section14c.Web/src/modules/pages/landingPageTemplate.html

```diff
@@ -2,26 +2,25 @@
 
 <main tabindex="-1" id="mainContent" role="main" class="main-content">
 
-  <div>
-    <main-top-nav-control></main-top-nav-control>
-    <div ng-show="applicationLoadError.status" role="alert" tabindex="0">
-      <div  class="dol-feedback-error">
-        <p class="dol-feedback-message">
-          <strong>We've run into a problem</strong>
-        </p>
-      </div>
-      <p>{{applicationLoadError.message}}</p>
-    </div>
-    <span ng-show="!organizations"> You do not have any employers associated with this account. Please click "Register a New Employer" to continue.</span>
-
-    <div class="container dol-employees-data-container">
-      <table id="EmployerTable" cclass="dol-datatable display responsive nowrap" cellspacing="0" width="100%" ng-show="organizations">
+  <main-top-nav-control></main-top-nav-control>
 
-      </table>
+  <div ng-show="organizations">
+    <div class="dol-application-start"> <!-- SHOW this div if application has not been started. HIDE this div if the application is in progress. -->
```

Comment:
Are these `SHOW` and `HIDE` comments reminders that this needs to be implemented?  If it's already done, I'm curious how it works.